methods.go

```diff
@@ -205,12 +207,72 @@ func decodeTorrentsStatusResponse(resp *DelugeResponse) (map[string]*TorrentStat
 	return result, nil
 }
 
+// TorrentError is a tuple of a torrent and an error message, returned by
+// methods that manipulate many torrents at once.
+type TorrentError struct {
+	// ID is the hash of the torrent that experienced an error
+	ID string
+	// Message is a string message
+	Message string
+}
+
+// Error satisfies the error interface
+func (t TorrentError) Error() string {
+	return fmt.Sprintf("<%s>: '%s'")
+}
+
+// RemoveTorrents tries to remove multiple torrents at once. If successful
+// an empty list is returned with a nil error.
+// If errors were encountered, the resulting list will be a list of
+// TorrentErrors.
+//
+// Will also try to delete all downloaded files for the torrents, If
+// `rmFiles` is set, otherwise they will need to be deleted manually later.
```

Comment:
Should specify what guarantees are there about file removal vs returned errors.

Reply:
I did my best to document how this method behaves, but it still is a little bit opaque to me. If you can see a better way to convey this information, please feel free to edit the wording :)

Reply:
I added that comment because it is unclear whether some or no files are deleted in case of failure; I understand one first has to understand how the underlying RPC API works to explain that.

In worst case it is fair to specify that in case of error the behaviour is undefined (but it is valuable to explicitly state so).

Reply:
Looking at https://github.com/deluge-torrent/deluge/blob/2f1c008a26b50ab3487bd03bcabb39347d441f23/deluge/core/core.py#L593 it seems that the only error we can expect to receive is InvalidTorrentError, unless I'm missing something. So the deletion of files does not factor into the returned errors at all. I'll add this info to the documentation.

Reply:
Great, thanks! That will be helpful.